app/authorities/sufia/authorities/local/find_works.rb

```diff
@@ -0,0 +1,21 @@
+module Sufia::Authorities
+  class Local::FindWorks < Qa::Authorities::Base
+    attr_reader :q
```

Comment:
You are talking about the parameter to initialize.  I am talking about `attr_reader :q` that is never used. Additionally, Is there some reason you need to override initialize? Can  it be completely inherited?